net-im/fractal/Makefile

```diff
@@ -16,12 +16,14 @@ BUILD_DEPENDS=	cargo:lang/rust \
 		gmake:devel/gmake
 LIB_DEPENDS=	libdbus-1.so:devel/dbus \
 		libgmp.so:math/gmp \
-		libgspell-1.so:textproc/gspell
+		libgspell-1.so:textproc/gspell \
+		libgtksourceview-3.0.so:x11-toolkits/gtksourceview3
 
 GL_HASH=	cc46d6b9702ce5d0b1f3073f516a58c3
 
 USES=		gettext gnome meson pkgconfig python:3.5+,build ssl tar:xz
 USE_GNOME=	gtk30 cairo
+USE_GSTREAMER1=	bad yes
```

Comment:
hm, is it actually a compile time dep?

Reply:
I don't think so, may be a videoconf ability.